Corrected naming to Composite Application Map

SteveBurkettNZ opened the pull request:
Looks to have changed name from Multi-Role Application Map in the Previews blade. Also clarified where to find the Previews blade.

azurecla commented:
Hi __@CtrlDot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Jim-Parker commented:
@cherylmc Cheryl, can you take a look at this contribution and let me and the author know if this should be merged? Thanks!

cherylmc commented:
Hi! Thanks for taking the time to let us know how we can make this doc better. I've sent this over to the PM to verify. I always do that, just so the PM knows that there is a change and that he's on board with it. Once he validates, I'll comment and we can move forward.

yushwang commented:
Hi, Looks good. Please go ahead and accept the PR.

Thanks,
Yushun

cherylmc commented:
All good here as well. You can move forward to accept the PR.

SteveBurkettNZ committed:
Corrected naming to Composite Application Map

Looks to have changed name from Multi-Role Application Map in the Previews blade. Also clarified where to find the Previews blade.
